package.json

```diff
@@ -30,7 +30,6 @@
     "test:single": "cross-env NODE_CONFIG_DIR='./tests/e2e/config' BABEL_ENV=commonjs mocha --compilers js:babel-register",
     "update-deps": "npm install -g rimraf && rimraf node_modules && rimraf npm-shrinkwrap.json && npm install && npm shrinkwrap --dev",
     "build": "npm run uglify && npm run makepot && npm run sass",
```

Comment:
First feedback on build artifact produced:
 - .idea, phpstan.neon needs to be excluded (some folks are using JB IDEs and .idea sneaks into the build artifact)
 - vendor folder was missing, the build task should build PHP/JS production dependencies

Let's push the related changes in this PR.

Reply:
Hey @kalessil , about these:

* Ok to ignore these, i'd say these goes to the composer.json file.
* About vendor folder not being present, i didn't change how the build process was from before our work here. I suppose i should do a `composer install --production` before the `composer archive` command?

Thanks

Reply:
Hey @msaggiorato,
- Sound good
- Yep. I'd suggest also wrapping it into a dedicated command like `"build:deps": "rm -rf vendor && composer install --no-dev --optimize-autoloader",`

Reply:
Resolved